backend/api/api_deployment_views.py

```diff
@@ -74,28 +74,29 @@ def get(
         self, request: Request, org_name: str, api_name: str, api: APIDeployment
     ) -> Response:
         execution_id = request.query_params.get("execution_id")
-        include_metadata = request.query_params.get("include_metadata", False)
+        include_metadata = (
+            request.query_params.get(ApiExecution.INCLUDE_METADATA, "false").lower()
+            == "true"
+        )
         if not execution_id:
             raise InvalidAPIRequest("execution_id shouldn't be empty")
         response: ExecutionResponse = DeploymentHelper.get_execution_status(
             execution_id=execution_id
         )
-        if response.execution_status != CeleryTaskState.SUCCESS.value:
-            return Response(
-                {
-                    "status": response.execution_status,
-                    "message": response.result,
-                },
-                status=status.HTTP_422_UNPROCESSABLE_ENTITY,
-            )
-        if (
-            response.execution_status == CeleryTaskState.SUCCESS.value
-            and not include_metadata
-        ):
-            response.remove_result_metadata_keys()
+        response_status = status.HTTP_200_OK
+        if response.execution_status == CeleryTaskState.COMPLETED.value:
```

Comment:
While testing all the responses were 422 and noticed that the status was `COMPLETED` and not `SUCCESS`. @muhammad-ali-e @athul-rs Please take a look here if this is correct.

Reply:
@Deepak-Kesavan 
This this mentioned aren't we only looking for `success` status or `422` status code ? if there is some other response comes, will we be sending 200 since it is mentioned as default for response_status and only changed in case of 422?

- Didn't get the full context, is 422 coming in for all responses ?

Reply:
Yes @athul-rs . In the earlier code, everything was 422. `EXECUTING`, `COMPLETED` etc, because we were checking for `SUCCESS` and the status was `COMPLETED`. So, I changed the status check alone.

I am changing the status to 422 in the else part. To avoid any confusion, I will make 422 as default and assign 200 inside if condition.

Reply:
Think that will make more sense. :+1: